portfolio/src/components/Info.tsx

```diff
@@ -1,131 +0,0 @@
-import React, { Component } from 'react';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
-import Button from 'react-bootstrap/Button';
-import ButtonGroup from 'react-bootstrap/ButtonGroup';
-import skills from '../data/skills.json';
-import timeline from '../data/timeline.json';
-
-type Props = {
-
-}
-type State = {
-    frontend: { name: string, level: number }[],
-    backend: { name: string, level: number }[],
-    misc: { name: string, level: number }[],
-    updates: { date: string, event: string, details: string }[]
-}
-export default class Info extends Component<Props, State>{
-    constructor(props: Props) {
-        super(props);
-    }
-    componentWillMount() {
-        this.setState({
-            frontend: [],
-            backend: [],
-            misc: [],
-            updates: []
-        });
-    }
-    componentDidMount() {
-        console.log(skills);
-        this.setState({
-            frontend: skills.frontend,
-            backend: skills.backend,
-            misc: skills.misc,
-            updates: timeline
-        });
```

Comment:
These don't have to be states. The import of `skills` is not async as far as I know.